src/modules/shared/translations/en-us.yml

```diff
@@ -0,0 +1,28 @@
+#
+# This file is used for the internal Zendesk translation system, and it is generated from the extract-strings.mjs script.
+# It contains the English strings to be translated, which are used for generating the JSON files containing the translation strings
+# for each language.
+#
+# If you are building your own theme, you can remove this file and just load the translation JSON files, or provide
+# your translations with a different method.
+#
+title: "Copenhagen Theme Shared Translations"
+packages:
+  - "cph-theme-shared"
+parts:
+  - translation:
+      key: "cph-theme-error-boundary.title"
```

Comment:
Nit: In the other modules we used the package name as prefix for the keys, In this case, it should be something like `cph-theme-shared.error-boundary.title`.

We can also keep it as it is now, it would be just nicer to be consistent.

Reply:
I changed that because it is translation for all of the shared module and not only for ErrorBoundary.